<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClearPrivateDataDialog</class>
 <widget class="QDialog" name="ClearPrivateDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clear Private Data</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="filesCheckBox">
     <property name="toolTip">
      <string>Select to clear the list of recently opened files</string>
     </property>
     <property name="text">
      <string>Recently opened files</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="projectsCheckBox">
     <property name="toolTip">
      <string>Select to clear the list of recently opened projects and project related histories</string>
     </property>
     <property name="text">
      <string>Recently opened projects and project histories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="multiProjectsCheckBox">
     <property name="toolTip">
      <string>Select to clear the list of recently opened multi projects</string>
     </property>
     <property name="text">
      <string>Recently opened multi projects</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="debugCheckBox">
     <property name="toolTip">
      <string>Select to clear the debug histories</string>
     </property>
     <property name="text">
      <string>Debug histories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="shellCheckBox">
     <property name="toolTip">
      <string>Select to clear the shell histories</string>
     </property>
     <property name="text">
      <string>Shell histories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="vcsCheckBox">
     <property name="toolTip">
      <string>Select to clear the VCS related histories</string>
     </property>
     <property name="text">
      <string>Version Control System histories</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pluginsCheckBox">
     <property name="toolTip">
      <string>Select to clear the private data of plug-ins not covered above</string>
     </property>
     <property name="text">
      <string>Plug-in private data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ClearPrivateDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ClearPrivateDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
